plugins/modules/cluster_shutdown.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: cluster_shutdown
+
+author:
+  - Polona Mihalič (@PolonaM)
+short_description: Shutdown cluster.
+description:
+  - Shutdown cluster.
```

Comment:
Add to description if module shut down a single node, or all nodes in a cluster.

Reply:
In the documentation it says: "Shutdown the system.". And we have only one node in each cluster, so I can't really test if really all nodes do shut down. We should reconfigure our system to have one cluster with two nodes to be able to test this. But we can also trust the documentation :)

Reply:
Or we ask in slack, and then amend to description if needed.